components/Wrapped/slides/PlatinumCardSlide.tsx

```diff
@@ -0,0 +1,100 @@
+import $ from "@/utils/theme";
+import type { SlideProps, SlideOptions } from "../internals/slidesHelper";
+import Background from "../components/Background";
+import styles from "@/styles/PlatinumCardSlide.module.css";
+
+export default function PlatinumCardSlide({ data }: SlideProps) {
+  return (
+    <div
+      style={{
+        display: "flex",
+        flexDirection: "column",
+        flexGrow: 1,
+        justifyContent: "center",
+        alignItems: "center",
+        height: "100%"
+      }}
+    >
+      <h1
+        {...$.title({
+          // You can apply classes by daisy-chaining them after the $
+          // You can then add CSS properties like you
+          // normally would inside of this object
+          color: "white",
+          alignSelf: "start",
+          justifySelf: "flex-start"
```

Comment:
Let's have both the header & card centred with only a 8 to 16px gap between the two. Hope that makes sense? Can make a PR into this PR if it doesn't.